fuel-p2p/Cargo.toml

```
@@ -13,14 +13,18 @@ description = "Fuel client networking"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-trait = "0.1.52"
+bincode = "2.0.0-rc.1"
 futures = "0.3"
 futures-timer = "3.0"
 ip_network = "0.4"
 libp2p = "0.41"
+rand = "0.8.5"
```

Unless there's a specific reason
```suggestion
rand = "0.8"
```